### PR TITLE
feat/add library wheel git workflow master

### PR DESCRIPTION
**Pull request by aarmoa**
**Before submitting this PR, please make sure**:

- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/")

**A description of the changes proposed in the pull request**:
This PR adds a new GitHub workflow to automatically generate and store in the repository a wheel file for Hummingbot. This will be required by the plug-in repositories in Hummingbot Prime. Since they will not contain all Hummingbot source code, they will rely on Hummingbot library to run all validations.
We should merge the PR directly in CoinAlpha fork repository, to avoid sending changes to the community edition that are required just because of Hummingbot Prime.
We are using a separate file for the new workflow, and we are storing the generated wheel in a new `library` folder, to avoid collisions with the open source repository and to enable the easy merge of changes coming from the open source community edition.

**Tests performed by the developer**:
Tested the new functionality by running it in a personal fork of the Hummingbot repository.

**Tips for QA testing**:
No testing required.

[ch-24276]

### CODE REVIEW

File: .github/workflows/hb_library_build_workflow.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+          mv dist/hummingbot*linux*.whl library/
+      # Commit and push generated wheel file
+      - name: Commit wheel file
+        shell: bash
+        if: steps.program-changes.outputs.cache-hit != 'true' || steps.conda-dependencies.outputs.cache-hit != 'true'
+        run: |
+          git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
+          git config --local user.name "github-actions[bot]"
+          git add -f library/*
+          git commit -m "Build Hummingbot library wheel"
```
Comment (petioptrv) [inline .github/workflows/hb_library_build_workflow.yml:66]: Maybe we should indicate here and in `hb_library_build_workflow_mac.yml` that one is a commit for Linux wheel and the other is for MacOS

**Review by petioptrv — CHANGES_REQUESTED** [review]
One small comment

File: .github/workflows/hb_library_build_workflow.yml (hunk 1 above)
Comment (aarmoa) [reply .github/workflows/hb_library_build_workflow.yml]: Changed

**Review by petioptrv — APPROVED** [review]
The changes look great — especially unifying the files

**Comment by aarmoa** [comment]
These changes are no longer required. The HB library will be created in the HB Prime repository.

**Pull request closed without merging**